examples/auto_review_predictions.py

```diff
@@ -26,17 +30,12 @@
 wf_results = wflow.get_submission_results_from_ids(submission_ids)
 predictions = wf_results[0].predictions.to_list()
 
-# Set up reviewer and review predictions
-review_config = ReviewConfiguration(
-    field_config=[
-        {"function": "remove_by_confidence", "kwargs": {"conf_threshold": 0.90}},
-        {
-            "function": "accept_by_confidence",
-            "kwargs": {"conf_threshold": 0.98, "labels": ["Name", "Amount"]},
-        },
-    ]
-)
-auto_reviewer = AutoReviewer(predictions, review_config)
+# Set up review functions and review predictions
+functions = [
+    AutoReviewFunction(remove_by_confidence, kwargs={"conf_threshold": 0.90}),
```

Comment:
default to all labels; also call that out in docstring in examples and auto review file

Reply:
resolved